resources/tiers.md

```diff
@@ -0,0 +1,89 @@
+
+:warning: The team will still keep all officially maintained SDKs **working**. :warning:
+
+## Tier #1
+
+The SDKs in group #1 have absolute priority over the team's time. For example, issues are visited before issues are present in tier #2.
+It is worth remembering that this does not mean open issues are addressed and fixed promptly. Every issue needs time to be understood/confirmed/worked on and consequently closed.
+
+All new Meilisearch features that make sense for those SDKs will be available on the release day.
+
+## Tier #2
+
+Tier #2 SDKs are the emergent ones. The Integrations Team believes they can move further and are betting on them.
+In any case, since they bring less value than tier #1, the team will only spend the time required to keep these SDKs working and do the necessary maintenance code and code cleanup.
+
+Only in urgent cases where, for example, the SDK is no longer working issues from the SDKs in group #2 prioritized over Tier #1.
+
+## Tier #3
+
+The SDKs in group #3 have relatively low user adoption compared to groups #2 and #1. They are probably new or have a small community behind the tech stack in the first place.
+As with tier #2, SDKs in this group are prioritized only in critical cases, and they are not going to receive active improvements or maintance from the integrations team.
+
+This is a very special group because the Integrations Team really need help from the community here.
+
+
+## Tiers list
+
+Integration | Tier |
+-------------|------|
+Javascript | #1 |
+PHP | #1 |
+Instant Meilisearch | #1 |
+Python | #1 |
+Go | #1 |
+Strapi | #2 |
+Ruby | #2 |
+Rails | #2 |
+.NET | #2 |
+Rust | #2 |
+Symfony | #3 |
+docs-searchbar.js | #3 |
+Firebase | #3 |
+Dart | #3 |
+Java | #3 |
+Swift | #3 |
+Vuepress | #3 |
+Gatsby | #3 |
+
+## FAQ
+### How are the tiers organized?
+
+The method used is mainly manual and subjective, using previously collected data from the [telemetry](https://docs.meilisearch.com/learn/what_is_meilisearch/telemetry.html) and Github to generate the groups.
+
+Only 3 tiers were defined, but this number may change in the future, and this guide will be updated accordindgly.
+
+### Is it possible for an SDK to move up a level?
+
+Yes, fully possible! Especially if the team detects that more users are interested in and using it.
+Whenever a anomaly is detected, a change can be submitted to the `integration-guides` and the groups will be modified to better manage the team's time. The reassessment of the tiers will be carried out every release cycle (8 weeks).
+
+The other way around is also valid. A SDK can move to less important tiers. :shipit:
+
+### How to identify which tier an SDK belongs to?
+
+It is possible by accessing the README.md of each repository. A tier badge is present below the Meilisearch logo. The badge can have the following values: "tiers #1", "tiers #2" and "tiers #3". For example:
```

Comment:
I'm not convinced that adding the badges is a good idea. SDK's will still work with the latest versions of Meilisearch and most users will be satisfied with its current state. 

By warning them that this is a low priority SDK it might result in:
- User  fearing deprecation
- User complaining because they don't want their SDK in that tier list even if they would never user the missing features
- Give the impression that we are going backward in activity like a "cut in budget"

If something is missing, an issue exist in any case with the missing feature which reassure the user that we did not forget about the SDK, provide them a place to tell us they need it, and help them implement it if they want it sooner than later.